DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/_CentreCourseCard.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@using DigitalLearningSolutions.Data.Models.Courses
+@model CourseStatistics
+
+<div class="searchable-element nhsuk-panel card-with-buttons" id="@Model.CustomisationId-card">
+  <details class="nhsuk-details nhsuk-expander nhsuk-u-margin-bottom-0">
+    <summary class="nhsuk-details__summary">
+      <span class="nhsuk-details__summary-text searchable-element-title" id="@Model.CustomisationId-name" name="name">
+        @Model.CourseName
+      </span>
+    </summary>
+    <div class="nhsuk-details__text">
+
+    </div>
+  </details>
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-full nhsuk-u-margin-left-4">
+      <a class="nhsuk-button nhsuk-u-margin-right-2" role="button"
+         asp-controller="LearningMenu" asp-action="Index" asp-route-customisationId="@Model.CustomisationId">
+        Launch course

```

Comment:
This button takes the user to the LearningMenu, but if the admin user is not also currently logged in to a delegate account, that page will redirect them to the home page due to a lack of authorisation. If they are logged into a delegate account, the page then redirects them to the old site, which seemed to crash locally (I suspect due to a lack of Session cookie data).

Reply:
I think it's ok but worth looking out for - can you add a comment on ticket to test this is ok on UAT. :)